### PR TITLE
feat(tier4_rtc_msgs): add rtc msgs

### PR DESCRIPTION
**Pull request by rej55**
## Description

Add message definition files for request to cooperate.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code is properly formatted
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code is properly formatted
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by rej55** [comment]
@isamu-takagi @yabuta @tkhmy 
All checks have passed. Could you review this PR?

**Review by tkhmy — APPROVED** [review]
LGTM
@isamu-takagi how do you think?

File: tier4_rtc_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  set(CMAKE_CXX_EXTENSIONS OFF)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
```
Comment (isamu-takagi) [inline tier4_rtc_msgs/CMakeLists.txt:12]: [nits] These c++ flags are not used for msgs.

**Review by isamu-takagi — APPROVED** [review]
LGTM (Commented for cmake flags but it has no effect. So it can merge)

File: tier4_rtc_msgs/CMakeLists.txt (hunk 1 above)
Comment (rej55) [reply tier4_rtc_msgs/CMakeLists.txt:12]: Thank you for your comment.
It seems to be same as the other packages, so I'd like to merge once.

**Pull request merged**